.github/workflows/tests.yml

```diff
@@ -44,6 +44,8 @@ jobs:
         working-directory: ${{runner.workspace}}/build
         run: |
           cmake ${GITHUB_WORKSPACE}/tdms -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DCMAKE_BUILD_TYPE=${{ matrix.build_type }}
+        env:
+          CXXFLAGS: -Wall
```

Comment:
I wonder if this would be better in CMakeLists.txt, so local compilation also throws those warnings. Definitely agree that `-Werror` should only be used in CI

Reply:
Fair enough, I changed it to always build with `-Wall` :+1: